src/main/webapp/script.js

```diff
@@ -111,7 +111,7 @@ function gui() {
       .name('Roll (degrees)')
       .onChange(function(value) {
         for (let i = 0; i < rotationData.size; i++) {
-          rotationData.get(i).origin.rotation.z = THREE.Math.degToRad(value);
+          rotationData.get(i).origin.rotation.x = THREE.Math.degToRad(value);
         }
       });
 }
```

Comment:
its a little hard to read bc i the comments are vague and there are some magic numbers. could you add some description about the different shapes refer to on the page? For example what is the cylinder showing?